File: Program8.py

def DivisibleByFive(iNo):

	iAns = iNo % 5
	if(iAns == 0):
		return 1
	else:
		return 0
	


def main ():

	iValue = int(input("Enter Number \n"))
	
	iRet = DivisibleByFive(iValue)

	if(iRet == 0):
		print( iValue,"is not divisible by 5")
	else:
		print( iValue,"is divisible by 5")	



if __name__ == "__main__":
	main()

File: Program8.java
import java.util.*;

public class Program8 {


	public static int DivisibleByFive(int iNo)
	{
		int iAns = 0;

		iAns = iNo % 5;

		if(iAns == 0)
		{
			return 1;
		}
		else
		{
			return 0;
		}
	}

	public static void main(String[] args) {
		
		Scanner sobj = new Scanner(System.in);
		
		System.out.println("Enter the Number");
		
		int iValue = sobj.nextInt();
		
		int iRet = DivisibleByFive(iValue);

		if(iRet == 0)
		{
			System.out.println(iValue + " is not divisible by 5");
		}
		else
		{
			System.out.println(iValue + " is divisible by 5");
		}


		
		
				

	}

}